src/context/MessengerEvent.js

```diff
@@ -77,6 +77,15 @@ export type Postback = {
   referral?: Referral,
 };
 
+export type GamePlay = {
+  game_id: string,
+  player_id: string,
+  context_type: 'SOLO' | 'THREAD' | 'GROUP',
+  context_id: string,
+  score: number,
+  payload: string,
```

Comment:
Object?

Reply:
It's JSON string in the rawEvent, but we convert it to JS object in event.gamePlay getter.